src/transformers/trainer.py

```diff
@@ -212,6 +220,15 @@ class Trainer:
             containing the optimizer and the scheduler to use. Will default to an instance of
             :class:`~transformers.AdamW` on your model and a scheduler given by
             :func:`~transformers.get_linear_schedule_with_warmup` controlled by :obj:`args`.
+
+    Important accessors:
+
+        ``self.model`` - always points to a PretrainedModule subclass (i.e. inner `transformers` model)
```

Comment:
```suggestion
        ``self.model`` - always points to the core model. If using a transformers model, it will be a :class:`PreTrainedModel` subclass.
```
Some users have their own models they feed to `Trainer`.